CHANGELOG.md

```diff
@@ -2,6 +2,23 @@
 
 ## [Unreleased]
 
+## [0.4.9] - 2019-12-12
+
+### Minimum Supported Rust Version
+
+This release bumps the minimum compiler version to `1.31.0`. This was mainly needed for `cfg-if`,
+but between `1.16.0` and `1.31.0` there are a lot of language and library improvements we now
+take advantage of.
```

Comment:
I think this should be `0.5.0` as bumping the minimum rust version is a breaking change, and `0.4.9` is considered semver-compatible — which means this release might break people's builds.

Reply:
We just have to be especially careful about breaking bumps in the `log` crate, because we have to consider compatibility between different versions of the `Log` trait.

Reply:
> bumping the minimum rust version is a breaking change

Just a note that this is not a unanimously held viewpoint in the ecosystem. e.g., `cfg-if` bumped its MSRV in a semver compatible release. There has been a lot of discussion on this point here: https://github.com/rust-lang/api-guidelines/issues/123